### PR TITLE
EDU-1177 Add cron job to delete old notifications

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1177

### CODE REVIEW

File: src/services/notification-service.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
+import moment from 'moment';
 import NotificationStore from '../stores/notification-store.js';
 import { groupNotifications } from '../utils/notification-utils.js';
 
+const MAX_NOTIFICATION_LIFETIME_IN_MONTHS = 6;
```
Comment (adrianaluchian) [inline src/services/notification-service.js:5]: We could put this in `constants.js` to make it easier to find, alongside the other such setup.

Comment (ahelmberger) [reply src/services/notification-service.js]: Yep, moved!

**Comment by ahelmberger** [comment]
I reworked this to just simply use a MongoDB TTL index (but kept the CronJob scheduler alive)

**Comment by adrianaluchian** [comment]
> I reworked this to just simply use a MongoDB TTL index (but kept the CronJob scheduler alive)

Perfect! 👌

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**